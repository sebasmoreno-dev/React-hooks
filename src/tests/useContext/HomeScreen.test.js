import { mount, shallow } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../../components/09-useContext/HomeScreen';
import { UserContext } from '../../components/09-useContext/UserContext';
import './../../setupTest';


describe('Pruebas en HomeScreen', () => {

  const user = {
    name: 'Juan',
    email: 'sebastian@gmail.com'
  }

  const wrapper = mount(
    <UserContext.Provider value={{
      user
      }}>

      <HomeScreen />
    </UserContext.Provider>
  );


  test('should show corretly' , () => {

    expect( wrapper ).toMatchSnapshot();

  });


});