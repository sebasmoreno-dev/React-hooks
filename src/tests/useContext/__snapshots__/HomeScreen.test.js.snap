// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HomeScreen should show corretly 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Juan",
   "email": "sebastian@gmail.com"
}
  </pre>
</div>
`;
